03-transactions.md

```diff
@@ -827,6 +836,102 @@ at each bucket is a prefix of the desired index.
 
 # Appendix A: Expected Weights
 
+## Expected Weight of the Funding Transaction (v2 Channel Establishment)
+
+The *expected weight* of a funding transaction is calculated as follows:
+
+      inputs: 40 bytes + var_int + `scriptlen`
+        - previous_out_point: 36 bytes
+          - hash: 32 bytes
+          - index: 4 bytes
+        - var_int: ? bytes (dependent on `scriptlen`)
+        - script_sig: `scriptlen`
+        - witness <---- Cost for "witness" data calculated separately.
+        - sequence: 4 bytes
+
+      non_funding_outputs: 8 bytes + var_int + `scriptlen`
+        - value: 8 bytes
+        - var_int: ? bytes (dependent on `scriptlen`)
+        - script: `scriptlen`
+
+      funding_output: 43 bytes
+        - value: 8 bytes
+        - var_int: 1 byte
+        - script: 34 bytes
+          - OP_0: 1 byte
+          - PUSHDATA(32-byte-hash): 33 bytes
```

Comment:
the funding output can also be a P2TR (it is the same size though)

Reply:
No, the funding output is P2WSH until the spec changes, at which point we will update this.